Comments/ClientApp/src/components/CommentList/CommentList.js

```diff
@@ -104,25 +104,36 @@ export class CommentList extends Component<PropsType, StateType> {
 		);
 
 		if (preloadedCommentsForCurrentUser) {
-			let allowComments = preloadedCommentsForCurrentUser.consultationState.consultationIsOpen && !preloadedCommentsForCurrentUser.consultationState.submittedDate;
+			const { comments, questions, consultationState } = preloadedCommentsForCurrentUser;
+			const {
+				consultationIsOpen,
+				submittedDate,
+				shouldShowDrawer,
```

Comment:
since it's no longer used, do you think you could get rid of shouldShowDrawer please?
it'll need removing from the backend too.